Documentation/cmdref/cilium-bugtool.md

```diff
@@ -37,9 +37,11 @@ cilium-bugtool [OPTIONS]
 
 ```
       --archive                 Create archive when false skips deletion of the output directory (default true)
+      --archive-prefix string   String to prefix to name of archive if created (e.g., with cilium pod-name)
```

Comment:
The change that introduced these options was not backported; I'm confused as to why `make -C Documentation/ cmdref` adds them. I checked out v1.2 and ran said command and it indeed introduces them. This may be a bug with how we generate the `cmdref`? I am going to build cilium based off of v1.2 and see if `make -C Documentation/ cmdref` exhibits the same behavior. The `cmdref` Makefile target appears to make it from the currently running cilium binary:

```
cmdref:
        $(QUIET) # We don't know what changed so recreate the directory
        $(QUIET) -rm -rvf $(CMDREFDIR)/cilium*
        @$(ECHO_GEN)cmdref/cilium
        $(QUIET) ${CILIUMDIR}/cilium cmdref -d $(CMDREFDIR)
        @$(ECHO_GEN)cmdref/cilium-bugtool
        $(QUIET) ${BUGTOOLDIR}/cilium-bugtool cmdref -d $(CMDREFDIR)
        @$(ECHO_GEN)cmdref/cilium-agent
        $(QUIET) ${AGENTDIR}/cilium-agent --cmdref $(CMDREFDIR)
        @$(ECHO_GEN)cmdref/cilium-health
        $(QUIET) ${HEALTHDIR}/cilium-health --cmdref $(CMDREFDIR)
```

So I think that if you built Cilium off of master, then checked out v1.2, and ran `make -C Documentation/ cmdref` it would generate the `cmdref` for master, not v1.2. I'll confirm this suspicion. I suspect that the build will fail because of this.

Reply:
You are absolutely right. I'm pretty sure that I ran `make` but I ran it again and the diff is different now.